### ISSUE TITLE
Deprecation warning due to invalid escape sequences

### ISSUE THREAD

**Issue by tirkarthi** [mozilla/mozdownload#609]
Deprecation warnings are raised due to invalid escape sequences. This can be fixed by using raw strings or escaping the literals. pyupgrade also helps in automatic conversion : https://github.com/asottile/pyupgrade/

```
find . -iname '*.py' | grep -Ev 'vendor|example|doc|tools|sphinx' | xargs -P4 -I{} python3.8 -Wall -m py_compile {}
./setup.py:21: DeprecationWarning: invalid escape sequence \d
  return re.findall("__version__ = '([\d\.]+)'",
```

**Comment by whimboo** [comment]
Thank you for the report. @tirkarthi, would you be also interested to get this issue fixed?

**Comment by tanmoydas9166** [comment]
Can I fix this issue?

**Comment by tirkarthi** [comment]
@tanmoydas9166 Feel free to pick it up.

**Comment by tanmoydas9166** [comment]
Sir, please check my PR.

**Comment by whimboo** [comment]
@tanmoydas9166 thanks for picking up this issue. I had a look at open PRs but sadly I cannot find the one from you. Did you actually create one? It's not enough to only push the changes to your own fork.

**Comment by paridhi7** [comment]
Hi, I'm new to open source. Is this issue still open?

**Comment by whimboo** [comment]
I haven't seen a PR being opened by @tanmoydas9166 yet. So feel free to pick-up this issue. Thanks.

**Comment by sahilsandhu** [comment]
Hi, i am new to open source, is this issue resolved or it is still opened??

**Comment by whimboo** [comment]
It's currently being worked on via PR #610. But maybe you want to help with https://github.com/mozilla/mozdownload/issues/247.

**Comment by rishishounak** [comment]
If the issue is still open could I join in

**Comment by whimboo** [comment]
If we don't hear back from @tanmoydas9166 within two days feel free to provide your own PR. Thanks!

**Comment by rishishounak** [comment]
I need some guidance to get started as I am new

**Comment by whimboo** [comment]
@rishishounak please check https://github.com/mozilla/mozdownload/pull/610 for the changes and my review comments. Those will be the lines of code you will have to change. Once done submit a PR. Thanks.

**Comment by rishishounak** [comment]
Please whether what I have done is correct

**Comment by rishishounak** [comment]
Please check whether I have made the right changes

**Comment by tanmaysinghal98** [comment]
@whimboo I am a first timer, can I pick this up?

**Comment by rishishounak** [comment]
@whimboo please check my pr

**Comment by whimboo** [comment]
Fixed by #612.
